python/ray/tune/integration/keras.py

```diff
@@ -45,4 +44,7 @@ def on_epoch_end(self, batch, logs={}):
         for metric in list(logs):
             if "loss" in metric and "neg_" not in metric:
                 logs["neg_" + metric] = -logs[metric]
-        self.reporter(keras_info=logs, mean_accuracy=logs["acc"])
+        if "acc" in logs:
```

Comment:
How does this thing depend on tf version?

Reply:
for some reason keras changes their result dictionary when moving to tensorflow.keras.